espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/GetAttribute.kt

```diff
@@ -32,11 +39,70 @@ class GetAttribute : RequestHandler<AppiumParams, String?> {
 
         // Map attributeName to ENUM attribute
         ViewAttributesEnum.values().find { attributeName.equals(it.toString(), ignoreCase = true) }
```

Comment:
the list of available attributes for espresso and compose is different, so I don't think we should unify that